tests/spec/core/xref-spec.js

```diff
@@ -5,6 +5,7 @@ import {
   makeDefaultBody,
   makeRSDoc,
   makeStandardOps,
+  xrefTestUrl,
```

Comment:
I think this function should be here only. Can't think of a use case to keep it in SpecHelper

Reply:
thought it would be nice because it's now used in two files

Reply:
(also, there is inline tests that should probably be in the inline-spec file... like the inline idl tests)

Reply:
Which other file?

Reply:
inlines-spec.js

Reply:
Yes. need to refactor these tests. xref-spec is too large

Reply:
In `inlines-spec.js`, we won't be making xref requests, right?

Reply:
https://github.com/w3c/respec/pull/2261/files#diff-e708af9e164f9fa08c467555607eee1cR7

Reply:
yes, for the  `[= For / thing  =]` tests, for example.